Install impossible

Unfortunately, I fail to install it according to the instructions on the website
```
sudo bash /tmp/feed.sh
Clone to '/usr/local/share/airplanes/git'...
remote: Enumerating objects: 17, done.
remote: Counting objects: 100% (17/17), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 17 (delta 2), reused 6 (delta 1), pack-reused 0 (from 0)
Empfange Objekte: 100% (17/17), 15.41 KiB | 751.00 KiB/s, fertig.
/usr/local/share/airplanes/git/configure.sh: Zeile 109: hostname: Command not found.
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
from https://github.com/airplanes-live/feed
 * branch            main       -> FETCH_HEAD
/usr/local/share/airplanes/git/update.sh: Zeile 164: adduser: Command not found.
/usr/local/share/airplanes/git/update.sh: Zeile 164: adduser: Command not found.
------------
[ERROR] Error in line 164 when executing: adduser --system --home-dir "$IPATH" --no-create-home "$UNAME"
```

--- a/configure.sh
+++ b/configure.sh
@@ -104,7 +104,9 @@ RECEIVERALTITUDE="$ALT"
 INPUT="127.0.0.1:30005"
 INPUT_TYPE="dump1090"
 
-if [[ $(hostname) == "radarcape" ]] || pgrep rcd &>/dev/null; then
+# `hostname` and `pgrep` (procps) are absent on some minimal images — guard both.
+HOSTNAME_VAL="$(hostname 2>/dev/null || uname -n 2>/dev/null || cat /etc/hostname 2>/dev/null || true)"
+if [[ "$HOSTNAME_VAL" == "radarcape" ]] || { command -v pgrep &>/dev/null && pgrep rcd &>/dev/null; }; then
     INPUT="127.0.0.1:10003"
     INPUT_TYPE="radarcape_gps"
 fi

--- a/update.sh
+++ b/update.sh
@@ -170,8 +170,12 @@ cp "$GIT"/scripts/*.sh "$IPATH"
 UNAME=airplanes
 if ! id -u "${UNAME}" &>/dev/null
 then
-    # 2nd syntax is for fedora / centos
-    adduser --system --home "$IPATH" --no-create-home --quiet "$UNAME" || adduser --system --home-dir "$IPATH" --no-create-home "$UNAME"
+    # Try Debian-style adduser, then Fedora-style adduser, then useradd.
+    # `||` chains are set -e safe; the trailing block makes the all-failed case explicit.
+    adduser --system --home "$IPATH" --no-create-home --quiet "$UNAME" \
+        || adduser --system --home-dir "$IPATH" --no-create-home "$UNAME" \
+        || useradd --system --home-dir "$IPATH" --no-create-home "$UNAME" \
+        || { echo "ERROR: failed to create user '$UNAME' (no working adduser/useradd)." >&2; exit 1; }
 fi
 
 echo 4

